src/Sfa.Tl.Matching.Web/Startup.cs

```diff
@@ -181,7 +181,8 @@ private static void RegisterRepositories(IServiceCollection services)
             services.AddTransient<IRepository<QualificationRoutePathMapping>, QualificationRoutePathMappingRepository>();
             services.AddTransient<IRepository<Route>, GenericRepository<Route>>();
             services.AddTransient<IRepository<Path>, GenericRepository<Path>>();
-            services.AddTransient<IRepository<Provider>, GenericRepository<Provider>>();
```

Comment:
--This is for Tech Debt -- 
check if there is any reason why we have registration as 
IRepository<Opportunity>, OpportunityRepository and Not as
IOpportunityRepository, OpportunityRepository

Reply:
Added as a note on description of https://skillsfundingagency.atlassian.net/browse/TLWP-638 (Inconsistent use of IRepository and ISpecificRepository)